README.md

```diff
@@ -364,6 +386,29 @@ to find instruction and links to exact configuration files and final checkpoints
 |MaskRCNN|INT8 (per-tensor for weights)|COCO2017|bbox: 37.27 (0.06)<br/>segm: 33.54 (0.02)|
 |MaskRCNN|Sparsity 50% (Magnitude)|COCO2017|bbox: 36.93 (0.40)<br/>segm: 33.23 (0.33)|
 
+### ONNX models
```

Comment:
It looks like ONNX example is out of place. In whole, the usage section requires reword, probably, need to separate it on demonstrating the config based approach and the pytonic approach... It requires discussion.

Reply:
I wanted to expose some results to users. So, I placed the table here. May you suggest better location?